heat/classification/knn.py

```diff
@@ -0,0 +1,50 @@
+import heat as ht
+
+
+class KNN(ht.ClassificationMixin, ht.BaseEstimator):
+    """
+    HeAT implementation of the K-Nearest-Neighbours Algorithm.
+
+    This algorithm predicts labels to data vectors by using an already labeled training dataset as reference.
+    The input vector to be predicted is compared to the training vectors by calculating the distance.
+    We use the euclidean distance, this can be expanded to other distance functions as well.
+    Then a majority vote of the k nearest (smallest distances) training vectors labels is chosen as prediction.
+
+    Parameters
+    ----------
+    x : ht.DNDarray
```

Comment:
Remove the ``required``.

Please check the capitalization of words (start of sentence, nouns lower case).

Reply:
The required are still in